packages/jss/src/SheetsRegistry.js

```diff
@@ -56,7 +56,7 @@ export default class SheetsRegistry {
   /**
    * Convert all attached sheets to a CSS string.
    */
-  toString({attached, ...options}: {attached?: boolean, ...ToCssOptions} = {}): string {
+  toString({attached, ...options}: {|attached?: boolean|} & ToCssOptions = {}): string {
```

Comment:
Why not make ToCssOptions exact and revert to type spread?

Reply:
try it, it will show up with other errors and this seemed to be the correct choice

Reply:
Inexact type is almost never correct choice. It's a hack for javascript dynamism.
This will solve the problem with exact type. `{...null}` is sealed object, can be shaped with explicit type annotation.
```
  options: ToCssOptions = {...null}

```

Reply:
this will add a bunch of stuff to the build I can not afford in a lib

Reply:
Only one case. Which stuff?

Reply:
```
    _proto.toString = function toString(_temp) {
      var _ref = _temp === void 0 ? _extends({}, null) : _temp,
          attached = _ref.attached,
          options = _objectWithoutPropertiesLoose(_ref, ["attached"]);
```
vs

```
    _proto.toString = function toString(_temp) {
      var _ref = _temp === void 0 ? {} : _temp,
          attached = _ref.attached,
          options = _objectWithoutPropertiesLoose(_ref, ["attached"]);
```

Reply:
🤷‍♂️

Reply:
Cast empty object to `any` then.

Reply:
any other options without additional runtime overhead?